README.rst

```diff
@@ -202,7 +202,7 @@ for details.
 
 **B905**: ``zip()`` without an explicit `strict=` parameter set. ``strict=True`` causes the resulting iterator
 to raise a ``ValueError`` if the arguments are exhausted at differing lengths. The ``strict=`` argument
-was added in Python 3.10, so don't enable this flag for code that should work on <3.10.
+was added in Python 3.10, so the check is disabled when running on python <3.10.
 For more information: https://peps.python.org/pep-0618/
```

Comment:
I think this should still warn that you shouldn't enable this if you want your code to work on <3.10, even if flake8 is ran on python version 3.10+

Reply:
Any suggestions about the text?